README.md

```diff
@@ -262,36 +262,25 @@ graph TB;
   R-. a,b,c prefer L .->L
 ```
 
-If a member `b` in `R` adds a new member `e` to `R`, then `b` MUST add `e`
-to `L` as soon as `b` detects the existence of `L` (figure 3).
-
-```mermaid
----
-title: Figure 3
----
-graph TB;
-  zero[X: a,b,c,d]
-  zero--"a excludes d"-->L[L: a,b,c]
-  zero--"b excludes d, adds e"-->R[R: a,b,c,e]
-  R-. a,b,c prefer L .->L
-  L-. b adds e .->L2[L: a,b,c,e]
-```
-
 Here we addressed two forked epochs.  In the generalized case where two or more
-forked epochs have the same `common` membership, then the tie-breaking rule
-(section 4.3.) is used to select the preferred epoch.  The tie-breaking rule
-supports multiple inputs.
+forked epochs have the same membership, then the tie-breaking rule (section 4.3.)
+is used to select the preferred epoch.  The tie-breaking rule supports multiple 
+inputs.
 
 
 ### 4.5. Resolving forked epochs with subset membership
 
-Suppose there are two forked epochs `L` and `R`.  If `common(L,R) ⊂ common(R,L)`,
-then all peers in `common(L,R)` who detect the existence of both `L` and `R`
-MUST select `L` as the preferred epoch over `R`. See figure 4.
+Suppose there are two forked epochs `L` and `R`.  If their membership are different
+`members(L) ≠ members(R)`, but `common(L,R) ⊆ common(R,L)`, then all peers in 
+`common(L,R)` who detect the existence of both `L` and `R` MUST select `L` as the
+preferred epoch over `R`. (This represents the set of members who have witnessed
+the bifurcation and are in the smaller of the groups - i.e. the group that excluded
+the most other members).
+See figure 3.
```

Comment:
note `⊆` here .... I think this needs some rework now to cover the cases as : 
- common(L,R) ⊂ common(R,L)
- common(L,R) = common(R,L)